simSPI/linear_simulator/configs/betagal_starfile.cfg

```diff
@@ -0,0 +1,71 @@
+[general]
+    name = betagal_starfile
+            #str: name of the dataset folder
+    usegpu= False
+            #bool: (default: False) option to use GPU TODO: gpu option to be added
+    datasetSize= 100
+            #int: size of the dataset
+    chunks=10
```

Comment:
num_chunks or chunck_size

`chunks` seems like num_chunks, but the comments suggests chunck_size